### PR TITLE
Landing page for the 2016 DORA Report

### PR DESCRIPTION
**Pull request by nathenharvey**
* Add tests for 2016 report page
* Add tests for new redirects
* Refactor tests for `/publications` and add tests for each report
* Add a 2016 report landing page `/research/2016`
* Link to the new landing page from `/publications`
* Link to the new landing page from `/research/2017-and-earlier/`

Preview links:
* https://doradotdev-staging--pr706-drafts-off-pttjo3so.web.app/research/2016
* https://doradotdev-staging--pr706-drafts-off-pttjo3so.web.app/publications
* https://doradotdev-staging--pr706-drafts-off-pttjo3so.web.app//research/2017-and-earlier/

This supports #698

### CODE REVIEW

**Comment by dora-machine-user** [comment]
Cloud Build time: Wed Jul 31 03:01:22 UTC 2024
Preview this PR [with drafts on] at https://doradotdev-staging--pr706-drafts-on-acvs5xc6.web.app
Preview this PR [with drafts off] at https://doradotdev-staging--pr706-drafts-off-pttjo3so.web.app

File: hugo/content/research/2016/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The 2016 State of DevOps Report reveals that high-performing organizations significantly outperform low performers in terms of IT performance and employee engagement. The report emphasizes benefits of integrating quality into every stage of development. The report also highlights the benefits of an experimental approach to product development, which involves breaking down products into smaller batches, ensuring visibility into the workflow, and actively seeking customer feedback.
+
+The report explores the relationship between IT performance and employee engagement, emphasizing the positive correlation between high performance and employee Net Promoter Scores (eNPS). It underscores the importance of integrating security practices throughout the software development lifecycle and advocates for lean product management practices, emphasizing the value of small batches, visibility, and customer feedback.
+
+Finally, the report provieds calculations for potential savings and discussing how these savings can be reinvested to foster innovation and drive business value. This work was later expaned in [The ROI of DevOps Transformation](/research/2020/) whitepaper.
```
Comment (davidstanke) [inline hugo/content/research/2016/_index.md:16]: `provieds` --> `provides`

File: hugo/content/research/2016/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+research_year: "2016"
+type: research_archives
+layout: single
+---
+
+[![2016 State of DevOps Report](/research/2016/2016-state-of-devops-report.png)](2016-state-of-devops-report.pdf)
+
+The 2016 State of DevOps Report reveals that high-performing organizations significantly outperform low performers in terms of IT performance and employee engagement. The report emphasizes benefits of integrating quality into every stage of development. The report also highlights the benefits of an experimental approach to product development, which involves breaking down products into smaller batches, ensuring visibility into the workflow, and actively seeking customer feedback.
+
+The report explores the relationship between IT performance and employee engagement, emphasizing the positive correlation between high performance and employee Net Promoter Scores (eNPS). It underscores the importance of integrating security practices throughout the software development lifecycle and advocates for lean product management practices, emphasizing the value of small batches, visibility, and customer feedback.
```
Comment (davidstanke) [inline hugo/content/research/2016/_index.md:14]: I feel like these first two paragraphs have a lot of overlap. I think we could even delete the second paragraph without losing anything. (In that case, we'd want to remove the word "Finally" from the subsequent paragraph).

**Review by davidstanke — COMMENTED** [review]
LG -- a couple nits

File: hugo/content/research/2016/_index.md (hunk 1 above)
Comment (nathenharvey) [reply hugo/content/research/2016/_index.md]: Fixed in 805c7e21ecd7b65ab6d01843dd4cd3dc4b930087

File: hugo/content/research/2016/_index.md (hunk 2 above)
Comment (nathenharvey) [reply hugo/content/research/2016/_index.md]: Fixed in 805c7e21ecd7b65ab6d01843dd4cd3dc4b930087

**Review by davidstanke — APPROVED** [review]
LGTM

**Pull request merged**